npm/config/config.go

```diff
@@ -4,6 +4,8 @@ import "github.com/Azure/azure-container-networking/npm/util"
 
 const (
 	defaultResyncPeriod         = 15
+	defaultNetPolMaxBatches     = 100
```

Comment:
Any reason we cannot reuse the defaultApplyMaxBatches and defaultApplyInterval ?

Reply:
discussed offline to have separate threads/config for windows and Linux